### PR TITLE
Adding log link to binary dist

### PR DESCRIPTION
**Pull request by kgoldfeld**

### CODE REVIEW

**Comment by kgoldfeld** [comment]
@assignUser Not sure why the usual system checks did not run. Does something need to be updated?

**Comment by assignUser** [comment]
Huh, that's weird... I tried re-running a previous job to test that there is no issue with the tags for the runners or something but that seems fine. Just try and push another commit to trigger it again?

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## New features
 * Added the ability to generate data from a empirical distribution by using new functions `genDataDensity` and `addDataDensity`.
+* The *binary* and *binomial* distributions can now accomodate a "log" link.
```
Comment (assignUser) [inline NEWS.md:5]:
```suggestion
* The *binary* and *binomial* distributions can now accomodate a "log" link. 
```

**Comment by kgoldfeld** [comment]
Yes - checks have been triggered.

**Comment by kgoldfeld** [comment]
@assignUser Assuming it passes all the checks, are you OK with my merging this quite minor but useful change?

**Review by assignUser — APPROVED** [review]

**Pull request merged**